### PR TITLE
Add upgrade apply instructions

### PR DESCRIPTION
**Pull request by MaximilianMeister**

### CODE REVIEW

File: adoc/admin-updates.adoc [hunk 1]
```diff
@@ -101,6 +101,69 @@ The control plane consists of these components:
...
+skuba node upgrade apply --target <worker-node-ip> --user <user> --sudo
+----
+
+Now all of your cluster nodes are upgraded. You can verify this by running:
+----
+kubectl get nodes -o wide
+----
+
+To enable the automatic OS updates again, `ssh` to each of the nodes and enable the `skuba-update.timer` again:
+
```
Comment (mssola) [inline adoc/admin-updates.adoc]: For some reason, I get this result (notice the extra `+`):

2019-07-09-160015_1916x1033_scrot

This only happens in this case. Rendered with the latest `asciidoctor-pdf` binary.

File: adoc/admin-updates.adoc [hunk 2]
```diff
@@ -101,6 +101,68 @@ The control plane consists of these components:
...
+[NOTE]
+=====
+The following steps are temporary and will be integrated into the official product soon.
+=====
+
+Updates have to be applied on a per-node base, starting from the control plane, down to the worker nodes.
+
+. To prepare the update, `ssh` to each of the nodes and perform the following steps:
+.. Disable the `skuba-update.timer`.
```
Comment (mssola) [inline adoc/admin-updates.adoc]: Use colon instead of period, just like in the other items :wink:

File: adoc/admin-updates.adoc [hunk 3]
```diff
@@ -101,6 +101,68 @@ The control plane consists of these components:
...
+To enable the automatic OS updates again, `ssh` to each of the nodes and enable the `skuba-update.timer` again:
+----
+sudo systemctl enbable --now skuba-update.timer
+----
+
+[TIP]
+=====
+The upgrade via `skuba node upgrade apply` will
+
+* upgrade the containerized control plane
```
Comment (mssola) [inline adoc/admin-updates.adoc]: Capitalize items.

File: adoc/admin-updates.adoc [hunk 4]
```diff
@@ -101,6 +101,68 @@ The control plane consists of these components:
...
+
+Updates have to be applied on a per-node base, starting from the control plane, down to the worker nodes.
+
+. To prepare the update, `ssh` to each of the nodes and perform the following steps:
+.. Disable the `skuba-update.timer`:
++
+----
+sudo systemctl disable --now skuba-update.timer
+----
+.. Create a temporary lock on the kubernetes component system packages:
```
Comment (ereslibre) [inline adoc/admin-updates.adoc]: I didn't need to do this in my environment, I did `zypper in kubernetes-kubeadm=1.15.0` and it worked just fine, just had to confirm that I wanted to "break" `1.15.0` when asked by zypper and I had `kubeadm=1.15.0` and the rest on `1.14.1`.

If that works I think it's even easier than asking to create locks.

**Comment by r0ckarong** [comment]
@MaximilianMeister 

- https://github.com/MaximilianMeister/doc-caasp/blame/upgrade-node-apply/adoc/admin-updates.adoc#L83
What is the implication here? What exactly is outdated on the control plane?
- https://github.com/MaximilianMeister/doc-caasp/blame/upgrade-node-apply/adoc/admin-updates.adoc#L90
What does `tolerate` mean in this context?
- https://github.com/MaximilianMeister/doc-caasp/blame/upgrade-node-apply/adoc/admin-updates.adoc#L182
`they are configurable` for what? The next three scenarios? Please add some clarification.

**Comment by MaximilianMeister** [comment]
>     * https://github.com/MaximilianMeister/doc-caasp/blame/upgrade-node-apply/adoc/admin-updates.adoc#L90
>       What does `tolerate` mean in this context?

this is the result of the first iteration of our work, and not fully complete yet. we're going to enhance the output in the next sprint, and will update the docs as well. `tolerate` just means that if one node is too outdated, it's a blocker for the update. but we'll have to fix that in the next iteration

**Pull request merged**